web/src/main/kotlin/com/example/ajaxproject/dto/request/UserDTO.kt

```diff
@@ -1,6 +1,24 @@
 package com.example.ajaxproject.dto.request
 
+
+import com.example.ajaxproject.User
+
 data class UserDTO(
-    var email: String = "",
-    var password: String = ""
+    val id: String,
+    val email: String,
+    val password: String,
 )
+fun User.toUserRequest(): UserDTO {
+    return UserDTO(
+        id = this.id,
+        email = this.email,
+        password = this.password,
+    )
+}
```

Comment:
You can omit **this** in all your extension functions

Reply:
Ok , did it